README.md

```diff
@@ -45,3 +51,26 @@ When this is the case, be mindful of not creating collision with the automated [
 The `unstable` or `development` channel is a special static one that is meant to include latest build of the development OBS projects. Hence this is an static JSON only
 including the ISO and OS container for the current build in projects under or branched from `isv:Rancher:Elemental` in OBS. This channel is not automated or generated by
 any kind of continuous integration.
+
+## Elemental channel images
+
+The Elemental channel images are defined under `.obs/dockerfile/elemental-channel-image`. 
+This folder is actually used to feed [rancher/elemental-obs](https://github.com/rancher/elemental-obs)
+repository which includes the OBS sources for all Elemental components.
+
+## Changelog generation for images listed in channels
+
+1. Changelogs are precomputed and generated at build time, as part of the Dockerfile, only if the `updatesinfo.xml` files are present.
+1. The updatesinfo.xml file is parsed and computed using the [updatesparser](https://github.com/rancher/elemental-channels/blob/main/updatesparser/README.md) utility included in this git repository.
+1. There is a single changelog file in JSON format per image listed within the channel.
+1. Each change log file includes the updates up to its creation date and after the creation date of the previous image. Oldest image in the list includes all the updates up to its creation date.
+1. Changelogs are included within the channel image in `/changelogs` folder. Each file is named with the name of the imge it relates to (aka the ManagedOSVersion name) with the `.updates.log` extension.
+1. Within the image sources there is the needed script to checkout the updateinfo.xml files at OBS commit time and the script to eventually iterate over the images listed in channels and precompute the changelog per image at build time.
+
+## Channels update sequence
+
+The semi automated process to generate and update channels including chanegelogs is essentially the following:
+
+1. Within the current repository the `.refresh_channels.sh` script is used in GHA or manually to recreate the channel .json files and the \*.packages.tar files. GHA creates automatic PRs, merging them is a manual process.
+2. In [rancher/elemental-obs](https://github.com/rancher/elemental-obs) GHA there is job to keep track of latest changes in this repository and update `elemental-channel-image` OBS package accordingly. Creates automatic PRs, merging them is a manual process.
+3. To include updatesinfo.xml files in OBS a manual process is required. Call the `update.sh` script on a locak checkout with the appropriate product URL, this will download the updatesinfo.xml files which can then be committed as part of the sources.
```

Comment:
```suggestion
3. To include updatesinfo.xml files in OBS a manual process is required. Call the `update.sh` script on a local checkout with the appropriate product URL, this will download the updatesinfo.xml files which can then be committed as part of the sources.
```